app/controllers/chatfuel_controller.rb

```diff
@@ -7,14 +7,83 @@ def show
     if tc
       text = Text::Renderer.new(text_component: tc).render(:main_part)
 
-      @response = {
-        messages: [
-          { text: text.first(640)} # Messages for Facebook Messenger can only be 640 characters long. Source: https://developers.facebook.com/docs/messenger-platform/send-api-reference#request
-        ]
-      }
-      render json: @response
+      qa = tc.question_answers
+
+      #Test API Request: http://localhost:3000/chatfuel/1
+      if qa.exists?(1)
+          sendJsonResponse(text, 1, qa.find(1).question)
```

Comment:
Do you mean the method sendJsonResponse() why I use it in both cases of the if statement?